ql/src/java/org/apache/hadoop/hive/ql/lockmgr/HiveTxnManager.java

```diff
@@ -161,7 +161,7 @@ void replTableWriteIdState(String validWriteIdList, String dbName, String tableN
    * @throws LockException if there is no current transaction or the
    * transaction has already been committed or aborted.
    */
-  void rollbackTxn() throws LockException;
+  void rollbackTxn(Context ctx) throws LockException;
```

Comment:
this is a change in API, we should keep the original signature

Reply:
You mean retain this function - 
`void rollbackTxn() throws LockException;`
 As well as create a -
`void rollbackTxn(Context ctx) throws LockException;` ?

Reply:
Added `void rollbackTxn(Context ctx)` and retained the old signature. Done. 

However most of the places will use `void rollbackTxn(Context ctx)` from now on.

Reply:
true, however, Impala could still be using an old API